cases-SEP24/transaction.test.js

```diff
@@ -117,6 +117,9 @@ describe("Transaction", () => {
     });
     const moreInfo = await fetch(json.transaction.more_info_url);
     expect(moreInfo.status).toEqual(200);
+    expect(moreInfo.headers.get("content-type")).toEqual(
```

Comment:
If the keys are case insensitive this looks good. Have you tested that this passes when run on our reference server?

Reply:
I would also maybe check for `content-type` _containing_ `text/html` instead of using direct equality.

Reply:
> If the keys are case insensitive this looks good. 
Just tested and yes the keys are case insensitive

> Have you tested that this passes when run on our reference server?

I tested it with this server https://testanchor.stellar.org

Reply:
> I would also maybe check for `content-type` _containing_ `text/html` instead of using direct equality.

Changed here https://github.com/stellar/transfer-server-validator/pull/174/commits/6dcf4a5d2bc77885cafcb0ae0bd51d2fd532ca88